core/src/main/scala/Config.scala

```diff
@@ -532,16 +537,28 @@ object Config {
                                       stg: StoreOp ~> IO,
                                       logger: LoggingOp ~> IO): IO[List[Datacenter]] = {
 
+    def readDocker(kfg: KConfig): Infrastructure.Docker = {
+
+      val creds =
+        (kfg.lookup[String]("docker.user"),
+         kfg.lookup[String]("docker.password")
+        ).mapN((a,b) => Docker.Credentials(a,b))
+
+      /*
+      If a datacenter specific registry isn't specified fall back to Nelson's publish registry
+       */
+      val registry = kfg.lookup[String]("docker.registry") getOrElse kfg.require[String]("docker-registry")
```

Comment:
You need
```
           nomad {
             endpoint = "{{ .nomad.endpoint }}"
             timeout = {{ .nomad.timeout }}
             mhz-per-cpu = {{ .nomad.mhz_per_cpu }}
              docker {
                  registry = "{{ .docker_registry }}"
              }
              docker-registry = "{{ .docker_registry }}"
           }
```
in `core/src/main/resources/nelson/defaults.cfg`.

Reply:
@timperrett @adelbertc Thoughts? It's been a minute now and I don't really know what I was thinking here. This was `docker.host`, which is semantically inaccurate because it is Nomad's primary Docker registry rather than a Docker execution target. `docker.registry` makes more sense between the two to me, but shouldn't Nomad just be using `[datacenter].docker-registry`?